### DISCUSSION TITLE
Downvote for Discussions

### DISCUSSION THREAD

**Discussion by just1a-person** [community/community#3173] [Discussions]
If we can have an upvote button we might as well have a downvote button

*Please downvote this discussion if this feature gets implemented*

**Comment by evi-liu** [comment]
Thanks for the feedback! Would you want everyone to have downvoting capabilities or just select permissions?

**Reply by just1a-person** [reply]
![Issue Reactions](https://user-images.githubusercontent.com/75578443/105447404-21fed580-5c9a-11eb-9598-9670140ff19a.png)

If anyone can use the downvote reaction then everyone should be able to downvote discussions as well

**Reply by killshot13** [reply]
I second the motion by @itsme-alan to add downvoting to Discussions. 

However, to make it easier to spot trolls and perpetually negative individuals, if a downvote feature is added, then each user's overall upvote and downvote tally should be publicly visible somewhere on their profile. 

Or even just make it optionally visible, and the stigma built around "having something to hide" will be enough to raise a red flag and cause others to pay closer attention to certain individuals if/when necessary. 😏

**Comment by luispadron** [comment]
Wasnt this in one of the Discussions beta? Really looking forward to see this added back (:

**Comment by usmonster** [comment]
Added my upvote for a Downvote feature. This kind of "feature suggestion" collection mode is a great use case for it. Specifically, a view on downvotes will give more insight as to how many feature requests are controversial (a better, more direct heuristic than, say, number of comments), whereas only collecting upvotes will only tell "one side" of the story.

**Comment by andig** [comment]
See https://github.com/github/feedback/discussions/9356 for making it configurable

**Comment by volkantash** [comment]
Güzel görünüyor mu? (Does it look good?)
![resim](https://user-images.githubusercontent.com/25375593/147581619-3666167e-a0ae-43bb-8f6b-b06d5d687393.png)

**Comment by aparatext** [comment]
Lack of downvoting is a big let down for Discussions.

Issues tend to be for developers discussing important matters and are already moderated by closing of issues or priority labels.

Discussions however focus mainly on the project's community instead. Without aggressive moderation, we can observe that only volume, not quality determines what gets to the top.

New users seeing only a moderate volume of unanswered basic questions, move the more interesting discussions like new relevant research, ideas, and personal projects to social media like Reddit instead.

Discussions would be a perfect feature, if not for how everything mildly important gets lost in the endless stream of uninformed questions.
